PMBio/deeprvat
# Feature burden sample ids

# What
This PR creates an additional zarr file output of the sample ids during the compute burdens execution. The sample ids correspond to the samples of which of the burdens were calculated for.
Sample ids within the zarr file can be used for additional association testing analysis, as desired.

# Testing
1. Run pytest
2. Run the snakemake pipeline on the example data
3. Run association testing pipeline on real data to verify the new zarr file is created correctly
